ansible/roles-studentvm/studentvm-ocp4/defaults/main.yml

```diff
@@ -8,13 +8,16 @@ studentvm_ocp4_defaults:
   # Software Version defaults
 
   # https://mirror.openshift.com//pub/openshift-v4/x86_64/clients/ocp
-  oc_version: '4.3.8'
+  oc_version: '4.3.13'
```

Comment:
This is very latest & greatest. I thought we were going to stay N-1 or N-2?

Reply:
That is stable-4.3